src/main/java/seedu/nutrigoals/model/user/Height.java

```diff
@@ -0,0 +1,49 @@
+package seedu.nutrigoals.model.user;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.nutrigoals.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Height object
+ */
+public class Height {
+    public static final String VALIDATION_REGEX = "\\d+";
+    public static final String MESSAGE_CONSTRAINTS = "Height must take on a positive number that is not too large.";
+    public final String value;
+
+    /**
+     * Initialises a Height object and checks if the specified input is a valid height
+     * @param height
+     */
+    public Height(String height) {
+        requireNonNull(height);
+        checkArgument(isValidHeight(height), MESSAGE_CONSTRAINTS);
+        value = height;
+    }
+
+    /**
+     * Checks if input height is valid
+     * @param height
+     * @return height is a number and height is < 220
+     */
+    public static boolean isValidHeight(String height) {
```

Comment:
Maybe we could update the `VALIDATION_REGEX` to check  if the user's height is of a certain range? 🤔